Source/Initialization/WarpXInitData.cpp

```diff
@@ -105,6 +105,169 @@ WarpX::PostProcessBaseGrids (BoxArray& ba0) const
+      }
+    else if (WarpX::maxwell_solver_id == MaxwellSolverAlgo::CKC){
+      amrex::Print()<<"Maxwell Solver:        | ckc" <<"\n";
+    }
+    else if (WarpX::maxwell_solver_id == MaxwellSolverAlgo::ECT){
+      amrex::Print()<<"Maxwell Solver:        | ect" <<"\n";
+    }
+  #ifdef WARPX_USE_PSATD
+    // Print PSATD solver's configuration
+    if (WarpX::maxwell_solver_id == MaxwellSolverAlgo::PSATD){
+      amrex::Print()<<"Maxwell Solver:       | psatd\n";
+      }
+    if ((m_v_galilean[0]!=0) or (m_v_galilean[1]!=0) or (m_v_galilean[2]!=0)) {
+      amrex::Print()<<"                      | - Galilean \n"<<
+      "                      |  - v_galilean=("<<m_v_galilean[0]<<","<<m_v_galilean[1]<<","<<m_v_galilean[2]<<")\n";
+      }
+    if (WarpX::update_with_rho==1) {
+      amrex::Print()<<"                      | - update with rho\n";
+      }
+    if (WarpX::do_dive_cleaning==1) {
+      amrex::Print()<<"                      | - div(E) cleaning\n";
+      }
+    if (WarpX::do_divb_cleaning==1) {
+      amrex::Print()<<"                      | - div(B) cleaning\n";
+      }
+    if (do_multi_J == 1){
+        amrex::Print()<<"                      | - multi-J deposition\n";
+        amrex::Print()<<"                      |   - J-deposition number="<<WarpX::do_multi_J_n_depositions<<"\n";
+    }
+    if (fft_do_time_averaging == 1){
+      amrex::Print()<<"                      | - time-averaged \n";
+    }
+  #endif // WARPX_USE_PSATD
+
+  if (do_nodal==1){
+    amrex::Print()<<"                      | - nodal mode" <<"\n";
+  }
+  #ifdef WARPX_USE_PSATD
+    if ( (do_nodal==0) && (field_gathering_algo == GatheringAlgo::EnergyConserving) ){
+      amrex::Print()<<"                      | - staggered mode " <<"\n";
+    }
+    else if ( (do_nodal==0) && (field_gathering_algo == GatheringAlgo::MomentumConserving) ){
+      amrex::Print()<<"                      | - hybrid mode\n";
+      amrex::Print()<<"                      |   - field_centering_nox="<<WarpX::field_centering_nox<<"\n";
+      amrex::Print()<<"                      |   - field_centering_noy="<<WarpX::field_centering_noy<<"\n";
+      amrex::Print()<<"                      |   - field_centering_noy="<<WarpX::field_centering_noz<<"\n";
+      amrex::Print()<<"                      |   - current_centering_nox="<<WarpX::current_centering_nox<<"\n";
+      amrex::Print()<<"                      |   - current_centering_noy="<<WarpX::current_centering_noy<<"\n";
+      amrex::Print()<<"                      |   - current_centering_noy="<<WarpX::current_centering_noz<<"\n";
+    }
+    if (WarpX::use_hybrid_QED == true){
+      amrex::Print()<<"                      | - use_hybrid_QED = true" <<"\n";
+    }
+
+    // Print solver's order
+    amrex::Print()<< "Spectral order:       | - psatd.nox="<<nox_fft<<"\n";
+    amrex::Print()<< "                      | - psatd.noy="<<noy_fft<<"\n";
+    amrex::Print()<< "                      | - psatd.noz="<<noz_fft<<"\n";
```

Comment:
Here maybe to avoid the clarification "('-1' corresponds to infinite order)" one could also write the following:
```suggestion
    amrex::Print()<< "Spectral order:       | - psatd.nox = " << ((nox_fft == -1) ? "inf" : nox_fft) << "\n";
    amrex::Print()<< "                      | - psatd.noy = " << ((noy_fft == -1) ? "inf" : noy_fft) << "\n";
    amrex::Print()<< "                      | - psatd.noz = " << ((noz_fft == -1) ? "inf" : noz_fft) << "\n";
```
Not sure if this will compile, though: to be double checked before committing the suggestion.